lucille-core/src/test/java/com/kmwllc/lucille/connector/jdbc/DatabaseConnectorTest.java

```diff
@@ -218,18 +218,11 @@ public void testClose() throws ConnectorException, SQLException {
 
     // create the connector with the config
     DatabaseConnector connector = new DatabaseConnector(config);
-
     // call the execute method, then close the connection
     connector.execute(publisher);
-
-    Connection connection = connector.getConnection();
-
-    // verify that the connection has opened
-    assertFalse(connection.isClosed());
-
+    assertFalse(connector.isClosed());
     connector.close();
-
     // verify that the connection is actually closed
-    assertTrue(connection.isClosed());

```

Comment:
we should still check that the connection itself is closed

Reply:
connector.getConnection() no longer exists...  I'd have to add some getters to the class to get to the underlying list connections now.   I feel like isClosed() on the connector is probably ok.

Reply:
I think it's good not to have a getter that exposes the underlying connections. I think it would be better to have a test that checks that the connections are really closed though.

I think we can use the connection that dbHelper has open to inspect the number of sessions against the h2 database.

```
  public int checkNumConnections() throws SQLException {
    try(ResultSet rs = RunScript.execute(conn, new StringReader("select count(*) from information_schema.sessions;"))) {
      rs.next();
      return rs.getInt(1);
    }
  }
```
Should be 1 after connector.close() is called (1 being the connection owned by dbHelper)

Reply:
cool...  turns out DBTestHelper holds onto a connection from the beginning to the end of the unit tests.  So, I updated the tests to assert the number of connection is 1 before and after each of the test cases.